server/routes/incidentStatement/incidentStatement.ts

```diff
@@ -53,21 +54,28 @@ export default class IncidentStatementRoutes {
     if (error) return this.renderView(req, res, { error, incidentStatement, incidentStatementComplete })
 
     try {
-      await this.placeOnReportService.addOrUpdateDraftIncidentStatement(
+      const draftAdjudicationResult = await this.placeOnReportService.addOrUpdateDraftIncidentStatement(
         Number(id),
         incidentStatement,
         incidentStatementComplete === 'yes',
         user
       )
-      const pathname =
-        incidentStatementComplete === 'yes'
-          ? `/check-your-answers/${prisonerNumber}/${id}`
-          : `/place-the-prisoner-on-report/${prisonerNumber}/${id}`
+      const { draftAdjudication } = draftAdjudicationResult
+
+      const pathname = this.getNextPage(incidentStatementComplete === 'yes', draftAdjudication)
```

Comment:
FYI it's not great to have hard-coded strings like 'yesy', esp duplicated.

Better to read all user input as soon as possible and convert to data types (in this case a boolean)